cpp/src/reductions/segmented_all.cu

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2019-2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "cudf/types.hpp"
+#include <cudf/detail/reduction_functions.hpp>
+#include <cudf/detail/utilities/device_atomics.cuh>
+#include <reductions/simple_segmented.cuh>
```

Comment:
This is a local include. Note that `simple_segmented.cuh` is in the same folder as `segmented_*.cu`. Maybe we should simply write `#include "simple_segmented.cuh"`?

Reply:
I guess quotes would be preferred by this rule unless this header could be a “detail include”.

> Use quotes " to include local headers from the same relative source directory. This should only occur in source files and non-public header files. Otherwise use angle brackets <> around included header filenames.

— cuDF Developer Guide